src/unpacker/extraction_container.py

```diff
@@ -4,17 +4,21 @@
 import multiprocessing
 from contextlib import suppress
 from os import getgid, getuid
+from pathlib import Path
 from tempfile import TemporaryDirectory
 
 import docker
+import requests
 from docker.errors import APIError, DockerException
 from docker.models.containers import Container
 from docker.types import Mount
+from requests.adapters import HTTPAdapter, Retry
 
 from config import cfg
 
 DOCKER_CLIENT = docker.from_env()
 EXTRACTOR_DOCKER_IMAGE = 'fkiecad/fact_extractor'
+retries = Retry(total=5, backoff_factor=0.1)
```

Comment:
Why not put this directly in the argument? It is only used once.

Reply:
in-lined it